tmt/export.py

```diff
@@ -46,21 +47,26 @@ def export_to_nitrate(test, create, general):
     try:
         nitrate_id = test.node.get('extra-nitrate')[3:]
         nitrate_case = nitrate.TestCase(int(nitrate_id))
-        nitrate_case.summary # Make sure we connect to the server now
+        nitrate_case.summary  # Make sure we connect to the server now
         echo(style(f"Test case '{nitrate_case.identifier}' found.", fg='blue'))
     except TypeError:
         # Create a new nitrate test case
         if create:
             nitrate_case = create_nitrate_case(test)
             new_test_created = True
+            # Newly created tmt tests have special format summary
+            test._metadata['extra-summary'] = nitrate_case.summary
```

Comment:
Hm, this is a hidden side-effect, but I guess it doesn't matter much for CLI usage

Reply:
Yeah, this should be ok.